src/raftstore/store/config.rs

```diff
@@ -27,9 +27,9 @@ const RAFT_LOG_GC_THRESHOLD: u64 = 50;
 const RAFT_LOG_GC_COUNT_LIMIT: u64 = REGION_SPLIT_SIZE * 3 / 4 / 1024;
 const RAFT_LOG_GC_SIZE_LIMIT: u64 = REGION_SPLIT_SIZE * 3 / 4;
 const SPLIT_REGION_CHECK_TICK_INTERVAL: u64 = 10000;
-const REGION_SPLIT_SIZE: u64 = 64 * 1024 * 1024;
-const REGION_MAX_SIZE: u64 = 80 * 1024 * 1024;
-const REGION_CHECK_DIFF: u64 = 8 * 1024 * 1024;
+const REGION_SPLIT_SIZE: u64 = 96 * 1024 * 1024;
+const REGION_MAX_SIZE: u64 = REGION_SPLIT_SIZE + REGION_SPLIT_SIZE / 3;
```

Comment:
I think `REGION_SPLIT_SIZE / 4 * 5` is more appropriate. `REGION_SPLIT_SIZE` can be divided by 4 or 8 normally.